set-timeskip-duration.lua

```diff
@@ -0,0 +1,214 @@
+-- Change the duration of the "Updating World" timeskip preceding the start of a new game.
+-- Author: Atomic Chicken
+
+local usage = [====[
+
+set-timeskip-duration
+=====================
+Starting a new fortress/adventurer session is preceded by
+an "Updating World" process which is normally 2 weeks long.
+This script allows you to modify the duration of this timeskip,
+enabling you to jump into the game earlier or later than usual.
+
+You can use this at any point before the timeskip begins
+(for example, while still at the "Start Playing" menu).
+
+It is also possible to run the script while the world is updating,
+which can be useful if you decide to end the process earlier
+or later than initially planned.
+
+Note that the change in timeskip duration will persist until either
+    the game is closed,
+    the "-clear" argument is used (see below),
+    or it is overwritten by setting a new duration.
```

Comment:
Just so you know, I'm adding bullets to this list. The indentation gets rendered in the HTML version, but single line breaks don't, so these lines are all getting combined into a single paragraph.

![image](https://user-images.githubusercontent.com/3719547/119601599-7d131700-bdb7-11eb-9495-6a66882a557c.png)

(Checking the output of the docs build through CI in this repo is a bit harder than in the main DFHack repo, unfortunately.)